switchmap/poller/snmp/async_poller.py

```
@@ -0,0 +1,135 @@
+"""Asynchronous SNMP Poller module for switchmap-ng. """
+
+import asyncio 
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)